forester/Dockerfile

```
@@ -1,20 +1,22 @@
     libssl-dev \
     && rm -rf /var/lib/apt/lists/*
 
-COPY .. .
+COPY . .
+RUN cargo clean
 RUN cargo build --release --package forester
 
-FROM debian:12-slim
-RUN apt-get update && apt-get install -y ca-certificates libssl-dev && rm -rf /var/lib/apt/lists/*
+FROM debian:bookworm-slim
+RUN apt-get update && apt-get install -y ca-certificates libssl3 && rm -rf /var/lib/apt/lists/*
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)